src/listener/boosterPass/boosterPassReceiverLeft.ts

```diff
@@ -0,0 +1,21 @@
+import { Listener } from 'discord-akairo';
+import { GuildMember } from 'discord.js';
+import { BoosterPassModel } from '../../model/boosterPass';
+
+export default class BoosterPassReceiverLeft extends Listener {
+	constructor() {
+		super('boosterPassReceieverLeft', {
+			emitter: 'client',
+			event: 'guildMemberRemove',
+		});
+	}
+
+	async exec(member: GuildMember) {
+		const receivedBoosterPasses = await BoosterPassModel.getReceivedByMember(
+			member
+		);
+
+		if (receivedBoosterPasses.length > 0)
```

Comment:
I don't think this condition is necessary, you can just let it iterate over the empty array

Reply:
Does need to be checked if its not-null s before calling forEach though

Reply:
The function `getReceivedByMember` won't return null as performing a find query on the collection returns an array of documents rather than just one. If there is no documents found in the query, it just returns an empty array. This is why jellz is just saying to remove the if-statement and to let it iterate as it's guaranteed you'll get an array of documents, empty or not.